src/main/java/io/supertokens/storage/mysql/queries/BulkImportQueries.java

```diff
@@ -0,0 +1,303 @@
+import io.supertokens.storage.mysql.Start;
+import io.supertokens.storage.mysql.config.Config;
+import io.supertokens.storage.mysql.utils.Utils;
+
+public class BulkImportQueries {
+    static String getQueryToCreateBulkImportUsersTable(Start start) {
+        String tableName = Config.getConfig(start).getBulkImportUsersTable();
+        return "CREATE TABLE IF NOT EXISTS " + tableName + " ("
+                + "id CHAR(36),"
+                + "app_id VARCHAR(64) NOT NULL DEFAULT 'public',"
+                + "primary_user_id VARCHAR(64),"
+                + "raw_data TEXT NOT NULL,"
+                + "status VARCHAR(128) DEFAULT 'NEW',"
+                + "error_msg TEXT,"
+                + "created_at BIGINT NOT NULL, "
+                + "updated_at BIGINT NOT NULL,"
+                + "PRIMARY KEY (app_id, id),"
+                + "FOREIGN KEY (app_id)"
+                + " REFERENCES " + Config.getConfig(start).getAppsTable() + " (app_id) ON DELETE CASCADE"
+                + " );";
+    }
+
+    public static String getQueryToCreateStatusUpdatedAtIndex(Start start) {
+        return "CREATE INDEX bulk_import_users_status_updated_at_index ON "
+                + Config.getConfig(start).getBulkImportUsersTable() + " (app_id, status, updated_at)";
+    }
+
+    public static String getQueryToCreatePaginationIndex1(Start start) {
+        return "CREATE INDEX bulk_import_users_pagination_index1 ON "
+                + Config.getConfig(start).getBulkImportUsersTable() + " (app_id, status, created_at DESC, id DESC)";
+    }
+
+    public static String getQueryToCreatePaginationIndex2(Start start) {
+        return "CREATE INDEX bulk_import_users_pagination_index2 ON "
+                + Config.getConfig(start).getBulkImportUsersTable() + " (app_id, created_at DESC, id DESC)";
+    }
+
+    public static void insertBulkImportUsers(Start start, AppIdentifier appIdentifier, List<BulkImportUser> users)
+            throws SQLException, StorageQueryException {
+        StringBuilder queryBuilder = new StringBuilder(
+                "INSERT INTO " + Config.getConfig(start).getBulkImportUsersTable() + " (id, app_id, raw_data, created_at, updated_at) VALUES ");
+
+        int userCount = users.size();
+
+        for (int i = 0; i < userCount; i++) {
+            queryBuilder.append(" (?, ?, ?, ?, ?)");
+
+            if (i < userCount - 1) {
+                queryBuilder.append(",");
+            }
+        }
+
+        update(start, queryBuilder.toString(), pst -> {
+            int parameterIndex = 1;
+            for (BulkImportUser user : users) {
+                pst.setString(parameterIndex++, user.id);
+                pst.setString(parameterIndex++, appIdentifier.getAppId());
+                pst.setString(parameterIndex++, user.toRawDataForDbStorage());
+                pst.setLong(parameterIndex++, System.currentTimeMillis());
+                pst.setLong(parameterIndex++, System.currentTimeMillis());
```

Comment:
please try to be consistent with this between postgres and mysql. I'm fine if you want to remove default in postgress and add these 2 lines there.

Reply:
I have updated the postgres plugin to use the same code.